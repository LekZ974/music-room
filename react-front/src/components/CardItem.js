import React from 'react';

import { CardItem as NBCardItem } from 'native-base';

const CardItem = props => {
  return <NBCardItem {...props} />;
};
export default CardItem;
