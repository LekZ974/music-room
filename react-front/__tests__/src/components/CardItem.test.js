import React from 'react';
import renderer from 'react-test-renderer';
import CardItem from '../../../src/components/CardItem';

it('renders correctly', () => {
  const tree = renderer
    .create(<CardItem>Snapshot test with CardItem Component!</CardItem>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
