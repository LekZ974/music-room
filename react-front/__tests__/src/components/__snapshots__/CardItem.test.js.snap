// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "borderRadius": 2,
      "flexDirection": "row",
      "paddingBottom": 10,
      "paddingLeft": 15,
      "paddingRight": 15,
      "paddingTop": 10,
    }
  }
>
  Snapshot test with CardItem Component!
</View>
`;
